File: level_2/narrative4_frequent_integer_not_blocked/langs/compiler.py
def main():
    n = int(input())
    arr = list(map(int, input().split()))
    m = int(input())
    b_arr = list(map(int, input().split()))
    print(frequent_integer_not_blocked(n, arr, m, b_arr))

if __name__ == "__main__":
    main()

File: level_2/narrative4_frequent_integer_not_blocked/langs/compiler.c
int main() 
{
    int n; 
    scanf("%d", &n);
    
    int arr[n]; 
    for (int i = 0; i < n; i++) 
        scanf("%d", &arr[i]);
    
    int m; 
    scanf("%d", &m);
    
    int b_arr[m]; 
    for (int i = 0; i < m; i++) 
        scanf("%d", &b_arr[i]);
    
    printf( "%d\n", frequent_integer_not_blocked(n,arr, m, b_arr));
    
    return 0;
}